docs/tasks/tls/certificate-rotation.md

```diff
@@ -0,0 +1,82 @@
+---
+approvers:
+- jcbsmpsn
+- mikedanese
+title: Certificate rotation
+---
+
+{% capture overview %}
+This page shows how to enable and configure certificate rotation for the kubelet.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
```

Comment:
I'm not sure if these are useful for advanced tasks.

Reply:
Removed.